contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -364,6 +368,27 @@ private void getColumnHeaders(SchemaBuilder builder) {
     builder.buildSchema();
   }
 
+  /**
+   * This function verifies whether a given column name is already present in the projected schema.
+   * If so, it appends _n to the column name.  N will be incremented for every duplicate column
+   * @param columnName The original column
+   * @return The deconflicted column name
+   */
+  private String deconflictColumnNames(String columnName) {
+    Pattern pattern = Pattern.compile("_(\\d+)$");
+    Matcher matcher = pattern.matcher(columnName);
+    while (excelFieldNames.contains(columnName)) {
```

Comment:
Does case matter because `contains` is case sensitive - so if col1 is called 'Column' and col2 is called 'column' then this code won't rename col2.

Reply:
@pjfanning 
Good catch.  It turns out it does matter... fixed and updated UT.